gradle/libs.versions.toml

```diff
@@ -9,8 +9,8 @@ androidx-navigation = "2.5.3"
 androidx-test-ext = "1.1.5"
 androidx-webkit = "1.7.0"
 datastorePreferences = "1.0.0"
-ditto = "4.7.4"
-live-ditto-tools = "2.0.0"
+ditto = "4.8.0"
```

Comment:
Can we make this SDK dependency upgrade its own PR? That way it shows up easily in the generated changelog and is easier to revert if something breaks.

Also, we need to remember this is a breaking change in terms of SemVer and requires a major version bump when we next release

Reply:
PR updated